src/actions/actionTypes.js

```diff
@@ -3,6 +3,8 @@
 // TARGETS
 export const LOAD_TARGETS_SUCCESS = 'load_targets_success';
 export const LOAD_TARGETS_FAILED = 'load_targets_failed';
+export const ADD_TARGET_SUCCESS = 'add_target_success';
```

Comment:
As a convention, we capitalize the redux actions to match the constant name: https://redux.js.org/basics/actions

```suggestion
export const ADD_TARGET_SUCCESS = 'ADD_TARGET_SUCCESS';
```

Reply:
Is done!!